devel/doctest/Portfile

```diff
@@ -0,0 +1,18 @@
+PortSystem 1.0
+PortGroup cmake 1.0
```

Comment:
please use the latest cmake PG, which is version `1.1`

Reply:
oh indeed, thanks, done